challenges/lib/src/test/java/challenges/stacksandqueues/QueueTest.java

```diff
@@ -0,0 +1,57 @@
+package challenges.stacksandqueues;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class QueueTest {
```

Comment:
Great work with your tests! It's important to break each test down to test one method at a time.